app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,7 @@
       android:label="@string/faq_activity_label" />
     <activity
       android:name=".app.help.faq.faqsingle.FAQSingleActivity"
+      android:label="@string/faq_activity_label"
```

Comment:
Rename string to `faq_activity_title`

Reply:
Also update it other places where its used.

Reply:
Done. PTAL.